projects/plugins/jetpack-boost/composer.json

```diff
@@ -0,0 +1,100 @@
+{
+	"name": "automattic/jetpack-boost",
+	"description": "Boost your WordPress site's performance, from the creators of Jetpack",
+	"type": "library",
+	"license": "GPL-2.0-or-later",
+	"version": "1.1.0",
+	"authors": [
+		{
+			"name": "Automattic, Inc.",
+			"email": "support@jetpack.com"
+		}
+	],
+	"minimum-stability": "dev",
+	"prefer-stable": true,
+	"require": {
+		"ext-json": "*",
+		"automattic/jetpack-autoloader": "2.10.3",
+		"automattic/jetpack-config": "1.4.6",
+		"automattic/jetpack-connection": "1.29.0",
+		"automattic/jetpack-device-detection": "1.4.2",
+		"automattic/jetpack-lazy-images": "1.5.0",
+		"automattic/jetpack-terms-of-service": "1.9.8",
+		"automattic/jetpack-tracking": "1.13.7",
+		"tedivm/jshrink": "1.4.0"
+	},
+	"require-dev": {
+		"10up/wp_mock": "^0.3.0",
+		"dealerdirect/phpcodesniffer-composer-installer": "*",
+		"automattic/jetpack-codesniffer": "^2.2",
+		"guzzlehttp/guzzle": "7.3.x",
+		"mockery/mockery": "1.3.x-dev",
+		"phpunit/phpunit": "^6",
+		"yoast/phpunit-polyfills": "0.2.0",
+		"automattic/jetpack-changelogger": "^1.1",
+		"automattic/wordbless": "dev-master"
+	},
+	"scripts": {
+		"format": [
+			"phpcbf ."
+		],
+		"lint": [
+			"phpcs ."
+		],
+		"phpunit": [
+			"@composer install",
+			"./vendor/phpunit/phpunit/phpunit --colors=always"
+		],
+		"test-coverage": [
+			"@composer install",
+			"phpdbg -d memory_limit=2048M -d max_execution_time=900 -qrr ./vendor/bin/phpunit --coverage-clover \"$COVERAGE_DIR/clover.xml\""
+		],
+		"test-php": [
+			"@composer phpunit"
+		],
```

Comment:
Do you have any non-e2e JS tests? I didn't see any defined in package.json, but maybe I missed them.

If so, you'd want to create a "test-js" script to run them, and also run them with coverage from "test-coverage".